src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Cells/HealthCertificateQRCodeCell.swift

```diff
@@ -77,13 +77,15 @@ class HealthCertificateQRCodeCell: UITableViewCell, ReuseIdentifierProviding {
 		qrCodeImageView.layer.magnificationFilter = CALayerContentsFilter.nearest
 		stackView.addArrangedSubview(qrCodeImageView)
 
-		certificateCountLabel.font = .enaFont(for: .headline)
-		certificateCountLabel.numberOfLines = 0
-		stackView.addArrangedSubview(certificateCountLabel)
+		titleLabel.style = .headline
```

Comment:
Is this now a global thing or only at this place? I.e. can I use just `.headline` everywhere?

Reply:
We actually had this all the time already but only used it in xib files, I also just realized this a couple of days ago. Instead of setting the `font` we could just set the `style` directly. It is required now to use `style` if you want to use the new highlighting with `**` on the `ENALabel`, otherwise `font` still works, but we could replace it. I already have it on my list for refactoring ideas to do this.